google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/cloudpubsub/internal/AckSetTrackerImpl.java

```diff
@@ -24,120 +24,117 @@
 import com.google.cloud.pubsublite.Offset;
 import com.google.cloud.pubsublite.SequencedMessage;
 import com.google.cloud.pubsublite.internal.CheckedApiException;
-import com.google.cloud.pubsublite.internal.CloseableMonitor;
 import com.google.cloud.pubsublite.internal.ExtractStatus;
-import com.google.cloud.pubsublite.internal.TrivialProxyService;
+import com.google.cloud.pubsublite.internal.ProxyService;
 import com.google.cloud.pubsublite.internal.wire.Committer;
-import com.google.common.collect.ImmutableList;
+import com.google.common.flogger.GoogleLogger;
 import com.google.errorprone.annotations.concurrent.GuardedBy;
 import java.util.ArrayDeque;
 import java.util.Deque;
-import java.util.List;
 import java.util.Optional;
 import java.util.PriorityQueue;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class AckSetTrackerImpl extends ProxyService implements AckSetTracker {
+  private static final GoogleLogger LOGGER = GoogleLogger.forEnclosingClass();
 
-public class AckSetTrackerImpl extends TrivialProxyService implements AckSetTracker {
   // Receipt represents an unacked message. It can be cleared, which will cause the ack to be
```

Comment:
Nit; It can no longer be "cleared," you just increase the generation version.

Reply:
Done.